Userland/Libraries/LibMarkdown/TextAlignment.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2021, Dana Burkart <dana.burkart@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TextAlignment.h"
+
+namespace Markdown {
+
+String TextAlignment::justify(String text, int justification_width, bool ignore_terminal_sequences)
```

Comment:
This should really operate on `Markdown::Text`. It already splits by whitespace, and doesn't require unnecessary escape sequence parsing.